invenio/modules/tags/models.py

```diff
@@ -257,7 +257,8 @@ def __init__(self, bibrec=None, **kwargs):
         """TODO."""
         super(WtgTAGRecord, self).__init__(**kwargs)
 
-        self.bibrec = bibrec
+        if bibrec is not None:
```

Comment:
Thanks for the answer, I am sorry I have no idea though. Just I wasn't able to define how it worked by looking at the code and was just curious. 

If I get some time I will take a look at why tests are broken  !